#!/usr/bin/env python

PACKAGE = "zivid_ros_wrapper"
import roslib;
roslib.load_manifest(PACKAGE);
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("iris_1", int_t, 0, "Iris", 13, 0, 72)
gen.add("iris_2", int_t, 0, "Iris", 22, 0, 72)
gen.add("iris_3", int_t, 0, "Iris", 33, 0, 72)

exit(gen.generate(PACKAGE, "zivid_ros_wrapper", "TripleHDR"))
